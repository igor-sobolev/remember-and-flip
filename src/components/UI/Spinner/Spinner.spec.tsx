import React from 'react';

import { Spinner } from '.';
import { shallow, ShallowWrapper } from 'enzyme';

describe('Given Spinner component', () => {
  describe('when it is rendered', () => {
    let wrapper: ShallowWrapper;
    beforeEach(() => {
      wrapper = shallow(<Spinner />);
    });

    it('should match the snapshot', () => {
      expect(wrapper.getElement()).toMatchSnapshot();
    });
  });
});
