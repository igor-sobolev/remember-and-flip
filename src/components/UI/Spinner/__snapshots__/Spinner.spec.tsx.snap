// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given Spinner component when it is rendered should match the snapshot 1`] = `
<div
  className="loader"
/>
`;
